tools/bashreadline.bt

```diff
@@ -12,6 +12,32 @@
  * Copyright 2018 Netflix, Inc.
  *
  * 06-Sep-2018	Brendan Gregg	Created this.
+ *
+ * Demonstrations of bashreadline, the Linux bpftrace/eBPF version.
+ * 
+ * 
+ * This prints bash commands from all running bash shells on the system. For
+ * example:
```

Comment:
This basically repeats what is already stated above. How about we just simplify it to:
```suggestion
 * Example of usage:
```